### PR TITLE
Fix formatting issues in the data services guide

### PR DESCRIPTION
**Pull request by gabilang**
Fixes the misalignments and a missing image in the doc.

### CODE REVIEW

File: en/docs/develop/creating-artifacts/data-services/creating-data-services.md [hunk 1]
```diff
@@ -18,38 +18,37 @@ Follow the steps given below to create the data service file:
 
 2. Click on the `+` beside **Data Services** in the **Micro Integrator: Project Explorer** sidebar to create a new data service.
```
Comment (DinithiDiaz) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:19]:
```suggestion
2. In the **Micro Integrator: Project Explorer** sidebar, hover over **Data Services** and click the **+** icon that appears to create a new data service.
```

File: en/docs/develop/creating-artifacts/data-services/creating-data-services.md [hunk 2]
```diff
@@ -91,44 +90,44 @@ Use this section to configure an REST resource for invoking the data service.
...
-   <a href="{{base_path}}/assets/img/integrate/tutorials/data_services/open-operation-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/open-operation-graphical-view.png" width="70%"></a>
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/open-operation-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/open-operation-graphical-view.png" width="80%"></a>
 
-   The graphical view will appear as shown below. You can use it to configure the main query details.
+	The graphical view will appear as shown below. You can use it to configure the main query details.
 
-   <a href="{{base_path}}/assets/img/integrate/tutorials/data_services/soap-operation-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/soap-operation-view.png" width="70%"></a>  
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/soap-operation-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/soap-operation-view.png" width="80%"></a>  
 
 ### Creating queries for REST resources and SOAP operations
```
Comment (DinithiDiaz) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:124]:
```suggestion
### Create queries for REST resources and SOAP operations
```

File: en/docs/develop/creating-artifacts/data-services/creating-data-services.md [hunk 3]
```diff
@@ -91,44 +90,44 @@ Use this section to configure an REST resource for invoking the data service.
...
-	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/open-resource-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/open-resource-graphical-view.png" width="70%"></a>  
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/open-resource-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/open-resource-graphical-view.png" width="80%"></a>  
 
-   The graphical view will appear as shown below. You can use it to configure the main query details.
+	The graphical view will appear as shown below. You can use it to configure the main query details.  
 
-	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/resource-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/resource-graphical-view.png" width="70%"></a>  
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/resource-graphical-view.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/resource-graphical-view.png" width="80%"></a>  
 
 ### Adding a SOAP operation
```
Comment (DinithiDiaz) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:104]:
```suggestion
### Add a SOAP operation
```

File: en/docs/develop/creating-artifacts/data-services/creating-data-services.md [hunk 4]
```diff
@@ -16,40 +16,39 @@ Follow the steps given below to create the data service file:
...
 3.  Enter a name for the data service and click **Create**.
 
-A data service file (DBS file) will now be created in your data service
-project as shown below.
+	A data service file (DBS file) will now be created in your data service project as shown below.
 
-![]({{base_path}}/assets/img/integrate/tutorials/data_services/data-service-project-structure.png)
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/data-service-project-structure.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/data-service-project-structure.png" width="80%"></a>
 
 ### Adding a datasource
```
Comment (DinithiDiaz) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:33]:
```suggestion
### Add a datasource
```

File: en/docs/develop/creating-artifacts/data-services/creating-data-services.md [hunk 5]
```diff
@@ -175,117 +174,121 @@ You can configure input parameters for the query using this section.
...
+			</table>  
+
+		3.	Save the input mapping. You can add multiple input mappings as required.  
 
 	Shown below is an example query with input mapping:
 
-	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png" width="70%"></a>
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png" width="80%"></a>
 
 #### Result (Output Mappings)
```
Comment (DinithiDiaz) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:240]:
```suggestion
#### Result (output mappings)
```

**Review by DinithiDiaz — APPROVED** [review]

File: en/docs/develop/creating-artifacts/data-services/creating-data-services.md [hunk 6]
```diff
@@ -167,125 +166,129 @@ You can configure the main query details for both SOAP operations and REST resou
...
+	
+			<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png" width="80%"></a>
+
+      	2. Enter the following output element details.
+
+
+             <table>
+                 <tr>
+                     <th>Property</th>
+                     <th>Description</th>
```
Comment (DinithiDiaz) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:264]: Shouldn't this be `Value` instead of `Description`?

Comment (chanikag) [reply en/docs/develop/creating-artifacts/data-services/creating-data-services.md:264]: Yes let's make it as Value

Comment (chanikag) [inline en/docs/develop/creating-artifacts/data-services/creating-data-services.md:264]:
```suggestion
                     <th>Value</th>
```

**Comment by DinithiDiaz** [comment]
LGTM

**Review by chanikag — APPROVED** [review]

**Pull request merged**